docs/clients/aws/ec2/ec2.md

```diff
@@ -0,0 +1,262 @@
+
+```bash
+aws ec2 import-key-pair --key-name "promtail-ec2" --public-key-material fileb://~/.ssh/id_rsa.pub
+```
+
+Next we're going to create a [security group][security group], make sure to note the group id, we'll need it for the following command:
+
+```bash
+aws ec2 create-security-group --group-name promtail-ec2  --description "promtail on ec2" --vpc-id vpc-668d120f
+{
+    "GroupId": "sg-02c489bbdeffdca1d"
+}
+```
+
+Now let's authorize inbound access for SSH and [Promtail][promtail] server:
+
+```bash
+aws ec2 authorize-security-group-ingress --group-id sg-02c489bbdeffdca1d --protocol tcp --port 22 --cidr 0.0.0.0/0
+aws ec2 authorize-security-group-ingress --group-id sg-02c489bbdeffdca1d --protocol tcp --port 3100 --cidr 0.0.0.0/0
+```
+
+> You don't need to open those ports to all IPs as shown above you can use your own IP range.
+
+We're going to create an [Amazon Linux 2][Amazon Linux 2] instance as this is one of the most popular but feel free to use the AMI of your choice.
+
+To create the instance use the following command, make sure to note the instance id:
+
+```bash
+aws ec2 run-instances --image-id ami-016b213e65284e9c9 --count 1 --instance-type t2.micro --key-name promtail-ec2 --security-groups promtail-ec2
+```
+
+To make it more interesting later let's tag (`Name=promtail-demo`) our instance:
+
+```bash
+aws ec2 create-tags --resources i-041b0be05c2d5cfad --tags Key=Name,Value=promtail-demo
+```
+
+> Tags enable you to categorize your AWS resources in different ways, for example, by purpose, owner, or environment. This is useful when you have many resources of the same type—you can quickly identify a specific resource based on the tags that you've assigned to it. You'll see later, Promtail can transform those tags into [Loki labels][labels].
+
+Finally let's grab the public DNS of our instance:
+
+```bash
+aws ec2 describe-instances --filters "Name=tag:Name,Values=promtail-demo" --query "Reservations[].Instances[].NetworkInterfaces[].Association.PublicDnsName"
+```
+
+and start an SSH session:
+
+```bash
+ssh ec2-user@ec2-13-59-62-37.us-east-2.compute.amazonaws.com
+```
+
+## Setting up Promtail
+
+First let's make sure we're running as root by using `sudo -s`.
+Next we'll download, install and give executable right to [Promtail][promtail].
+
+```bash
+mkdir /opt/promtail && cd /opt/promtail
+curl -O -L "https://github.com/grafana/loki/releases/download/v1.5.0/promtail-linux-amd64.zip"
+unzip "promtail-linux-amd64.zip"
+chmod a+x "promtail-linux-amd64"
+```
+
+Now we're going to download the [promtail configuration][promtail configuration] file below and edit it, don't worry we will explain what those means.
+The file is also available on [github][config gist].
+
+```bash
+curl https://gist.githubusercontent.com/cyriltovena/d0881cc717757db951b642be48c01445/raw/9492bfaaa776d77ce70d6aa69df89ff71524ade5/promtail-ec2.yaml > ec2-promtail.yaml
```

Comment:
I think we should host this from the Loki repo, maybe create a directory for sample configs?, or don't reference this at all and make people copy and paste from the yaml in the doc

Reply:
I'll add it to the repo. I did this for the other tutorial.